### ISSUE TITLE
Make Tune Step Part of Band Stacking

### ISSUE THREAD

**Issue by w3ub** [ramdor/Thetis-2.9.0#71]
Not sure what others think, but I would like the Tune Step setting to be part of the Band Stacking. For example, I use one Band Stack for CW, one for phone, ect. But the Tune Step is independent of this currently, and it would be much more convenient for it to be set per Band Stack.

/Doug

**Comment by ramdor** [comment]
yeah interesting. It is possible at the moment to store tunestep per mode, which should work as your bandstacks will select the mode cw/phone etc. The option is in Options-1 at the bottom of the stack of check boxes.

R.

**Comment by w3ub** [comment]
Ah, I see that option now … it is amazing how much there is still to learn even after a few years.
Doug

**Comment by rz3qs** [comment]
In BandStack, you need to remember all the current settings. RX1AF, AGC Gain, etc. Relevant for FT8.
The same applies to the TX profile.

**Comment by w3ub** [comment]
Agreed, that lots of other items such are NR2 etc should be stored in each band stack.
Doug
